os.hpp

```diff
@@ -24,33 +24,46 @@
 
 // Define the user struct
 enum userID {sys, u1, u2};
-struct User {
-  userID id;
-  bool running;
+userID nextUser(userID user) {
+  switch(user) {
+    case sys: return u1; break;
+    case u1: return u2; break;
+    case u2: return sys; break;
+  }
+  return user;
+}
+struct Process {
+  int id;
+  int status;
   registers regs;
+  // list<int>* tlb;
```

Comment:
This can be removed. We can access the page table pointer through `[Process].regs.PTBR`